### ISSUE TITLE
Warn users about about incompatibility with adblockers and Brave browser

### ISSUE THREAD

**Issue by ivan-aksamentov** [nextstrain/nextclade#439]
We had issues with adblocker browser extensions and Brave browser breaking Nextclade. Privacy features don't treat Nextclade's client-side computation model well.

Nextclade performs a lot of calculations on the client side, and uses WebWorkers and WebAssembly, the technologies that adblockers don't like. They may break features and make Nextclade to consume more memory.

I was not able to start analysis in Brave on Android - it crashes with out of memory error.
Also related: https://github.com/nextstrain/nextclade/issues/438

Perhaps we can warn Brave users that their browser is not supported and to recommend them to use a Chromium-based browser or Firefox.

We could detect Brave using [this API](https://github.com/nextstrain/nextclade/issues/438) and display a reactstrap alert or a popup dialog. Detecting adblockers though is harder and, as they are built to specifically avoid detction.

Popups should not interfere with "What's new" dialog.

Also, this might be controversial, as Nextclade is built with privacy in mind. We don't track users or send data to a server (we don't have any server). So using browser and adblocker detection goes somewhat against this idea.

**Comment by corneliusroemer** [comment]
> Perhaps we can warn Brave users that their browser is not supported and to recommend them to use a Chromium-based browser or Firefox.

Brave *is* Chromium based. It basically just has an in-built adblocker called *Brave shields* enabled by default.

The solution could potentially be as simple as switching that off.
![Screenshot_20210614-090144.jpg](https://user-images.githubusercontent.com/25161793/121852192-67767a80-ccef-11eb-8b75-4b9b879dd29c.jpg)

**Comment by rneher** [comment]
I don't think there is a strong use case on mobile at the moment (who has sequences lying around on their device?)

As a first step, we could put a message into the error pop-up saying that adblockers etc might be responsible for errors.

**Comment by corneliusroemer** [comment]
I just screenshot on mobile because I was typing on mobile. The shield is the same on desktop.

I agree that mobile is the least of our worries at the moment.

@ivan-aksamentov is this issue mobile only? I'm not sure, I just wanted to clarify Brave intricacies.

**Comment by ivan-aksamentov** [comment]
@corneliusroemer No, this is about any browser incompatibilities that we might to warn users about

**Comment by corneliusroemer** [comment]
Just wanted to clarify because I felt there was a potential misunderstanding here between us 3.
